{"name": "输入格式错误的手机号，点击获取验证码，查看系统提示", "status": "failed", "statusDetails": {"message": "AssertionError: assert None == '验证码发送失败、请输入正确的手机号码'", "trace": "self = <case.test_sgin.Test_sgin object at 0x00000000039C5730>\nget_driver = (<selenium.webdriver.chrome.webdriver.WebDriver (session=\"451e654c9d04ce46a4ba9fb3d3cf6b96\")>, <pages.login_login.Logi...ges.job_search.JobSearch object at 0x0000000003635AC0>, <pages.looking_talent.LookTalent object at 0x0000000003635760>)\n\n    @allure.story(\"验证码功能测试\")\n    @allure.title(\"输入格式错误的手机号，点击获取验证码，查看系统提示\")\n    def test_sgin_05(self, get_driver):\n        \"\"\"输入格式错误的手机号，点击获取验证码，查看系统提示.断言系统提示和字体颜色\"\"\"\n        sgin = get_driver[2]\n        with allure.step(\"点击跳转到注册页\"):\n            sgin.click_sgin()\n        with allure.step(\"输入注册手机号\"):\n            sgin.input_user_gr(user=\"157\")\n        with allure.step(\"点击获取验证码\"):\n            sgin.click_code_gr()\n        with allure.step(\"断言系统是否弹出正确的异常提示\"):\n            t1 = sgin.alert_text()\n            _log.info(\"断言系统的提示-- %s == 验证码发送失败、请输入正确的手机号码\" % t1)\n>           assert t1 == \"验证码发送失败、请输入正确的手机号码\"\nE           AssertionError: assert None == '验证码发送失败、请输入正确的手机号码'\n\ncase\\test_sgin.py:90: AssertionError"}, "description": "输入格式错误的手机号，点击获取验证码，查看系统提示.断言系统提示和字体颜色", "steps": [{"name": "点击跳转到注册页", "status": "passed", "steps": [{"name": "点击跳转注册页面", "status": "passed", "start": 1652081066895, "stop": 1652081066947}], "start": 1652081066895, "stop": 1652081066947}, {"name": "输入注册手机号", "status": "passed", "steps": [{"name": "输入注册的手机号", "status": "passed", "parameters": [{"name": "user", "value": "'157'"}], "start": 1652081066947, "stop": 1652081067004}], "start": 1652081066947, "stop": 1652081067005}, {"name": "点击获取验证码", "status": "passed", "steps": [{"name": "点击获取验证码", "status": "passed", "start": 1652081067005, "stop": 1652081067054}], "start": 1652081067005, "stop": 1652081067054}, {"name": "断言系统是否弹出正确的异常提示", "status": "failed", "statusDetails": {"message": "AssertionError: assert None == '验证码发送失败、请输入正确的手机号码'\n", "trace": "  File \"E:\\ptest\\youlingrcPC\\case\\test_sgin.py\", line 90, in test_sgin_05\n    assert t1 == \"验证码发送失败、请输入正确的手机号码\"\n"}, "steps": [{"name": "查看登录页系统提示内容", "status": "passed", "start": 1652081067054, "stop": 1652081082724}], "start": 1652081067054, "stop": 1652081082724}], "attachments": [{"name": "log", "source": "b01d0784-e003-4105-bdec-37bd4b00ab2f-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "56675cbe-b7ab-4a4c-80d5-a139d6f64a14-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "537fbabe-0fb7-4970-8a12-519bd6684fc5-attachment.txt", "type": "text/plain"}], "start": 1652081066895, "stop": 1652081082725, "uuid": "34742e02-a1cc-4bf1-95c8-98060a442eb9", "historyId": "d09324eb742e46700a3f9b589d17b822", "testCaseId": "79aea845b9e8e0096a4331737c705302", "fullName": "case.test_sgin.Test_sgin#test_sgin_05", "labels": [{"name": "feature", "value": "用户注册"}, {"name": "epic", "value": "用户登录/注册测试"}, {"name": "story", "value": "验证码功能测试"}, {"name": "tag", "value": "flaky(reruns=3, reruns_delay=2)"}, {"name": "tag", "value": "@pytest.mark.usefixtures('quit_rc_login')"}, {"name": "parentSuite", "value": "case"}, {"name": "suite", "value": "test_sgin"}, {"name": "subSuite", "value": "Test_sgin"}, {"name": "host", "value": "PC-20190311CFCL"}, {"name": "thread", "value": "10636-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "case.test_sgin"}]}